src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h

```diff
@@ -460,37 +460,39 @@ typedef INT_PTR SSIZE_T, *PSSIZE_T;
-#define STATUS_CANCEL_THREAD_FAILED              STATUS_BASE + 0x00000019
-#define STATUS_THREAD_IS_NOT_JOINABLE            STATUS_BASE + 0x0000001a
-#define STATUS_DETACH_THREAD_FAILED              STATUS_BASE + 0x0000001b
-#define STATUS_THREAD_ATTR_INIT_FAILED           STATUS_BASE + 0x0000001c
-#define STATUS_THREAD_ATTR_SET_STACK_SIZE_FAILED STATUS_BASE + 0x0000001d
-#define STATUS_MEMORY_NOT_FREED                  STATUS_BASE + 0x0000001e
+#define STATUS_BASE                                     0x00000000
+#define STATUS_NULL_ARG                                 STATUS_BASE + 0x00000001
+#define STATUS_INVALID_ARG                              STATUS_BASE + 0x00000002
+#define STATUS_INVALID_ARG_LEN                          STATUS_BASE + 0x00000003
+#define STATUS_NOT_ENOUGH_MEMORY                        STATUS_BASE + 0x00000004
+#define STATUS_BUFFER_TOO_SMALL                         STATUS_BASE + 0x00000005
+#define STATUS_UNEXPECTED_EOF                           STATUS_BASE + 0x00000006
+#define STATUS_FORMAT_ERROR                             STATUS_BASE + 0x00000007
+#define STATUS_INVALID_HANDLE_ERROR                     STATUS_BASE + 0x00000008
+#define STATUS_OPEN_FILE_FAILED                         STATUS_BASE + 0x00000009
+#define STATUS_READ_FILE_FAILED                         STATUS_BASE + 0x0000000a
+#define STATUS_WRITE_TO_FILE_FAILED                     STATUS_BASE + 0x0000000b
+#define STATUS_INTERNAL_ERROR                           STATUS_BASE + 0x0000000c
+#define STATUS_INVALID_OPERATION                        STATUS_BASE + 0x0000000d
+#define STATUS_NOT_IMPLEMENTED                          STATUS_BASE + 0x0000000e
+#define STATUS_OPERATION_TIMED_OUT                      STATUS_BASE + 0x0000000f
+#define STATUS_NOT_FOUND                                STATUS_BASE + 0x00000010
+#define STATUS_CREATE_THREAD_FAILED                     STATUS_BASE + 0x00000011
+#define STATUS_THREAD_NOT_ENOUGH_RESOURCES              STATUS_BASE + 0x00000012
+#define STATUS_THREAD_INVALID_ARG                       STATUS_BASE + 0x00000013
+#define STATUS_THREAD_PERMISSIONS                       STATUS_BASE + 0x00000014
+#define STATUS_THREAD_DEADLOCKED                        STATUS_BASE + 0x00000015
+#define STATUS_THREAD_DOES_NOT_EXIST                    STATUS_BASE + 0x00000016
+#define STATUS_JOIN_THREAD_FAILED                       STATUS_BASE + 0x00000017
+#define STATUS_WAIT_FAILED                              STATUS_BASE + 0x00000018
+#define STATUS_CANCEL_THREAD_FAILED                     STATUS_BASE + 0x00000019
+#define STATUS_THREAD_IS_NOT_JOINABLE                   STATUS_BASE + 0x0000001a
+#define STATUS_DETACH_THREAD_FAILED                     STATUS_BASE + 0x0000001b
+#define STATUS_THREAD_ATTR_INIT_FAILED                  STATUS_BASE + 0x0000001c
+#define STATUS_THREAD_ATTR_SET_STACK_SIZE_FAILED        STATUS_BASE + 0x0000001d
+#define STATUS_MEMORY_NOT_FREED                         STATUS_BASE + 0x0000001e

```

Comment:
I think these should go into the utils/Include.h

Reply:
Ohh I see. Fixed.
Sorry I'm finding the include file structures little confusing.